.gitignore

```diff
@@ -1,5 +1,6 @@
 target
 .sbtopts
 project/.sbt
+.idea
```

Comment:
Please read the comment below the line you just added.

Reply:
Also, please check out the diff. This didn't really ignore anything. You'll have to add the trailing slash for this to be valid, but please not here.